refactoring method getBlock()

azhangggggg opened the pull request:
### Description

Renamed getBlock(String blockId) to getActiveBlock(String blockId) since it's only getting active block via its blockId. Making this refactor to prepare for a new function getHiddenBlock() for feature - adding hidden questions to pdf.

### Checklist

#### General

Read the full guidelines for PRs [here](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#creating-a-pull-request)

- [x] Added the correct label: < feature | enhancement | bug | dependencies | infrastructure | ignore-for-release | database >
- [x] Assigned to a specific person, `civiform/developers`, or a [more specific round-robin list](https://github.com/civiform/civiform/wiki/Technical-contribution-guide#adding-reviewers).
- [ ] Created unit and/or browser tests which fail without the change (if possible)
- [x] Performed manual testing (Chrome and Firefox if it includes front-end changes)
- [ ] Extended the README / documentation, if necessary

### Instructions for manual testing

If instructions are needed for manual testing by reviewers, include them here.

### Issue(s) this completes

Part of #6288

bion commented on server/app/services/applicant/ReadOnlyApplicantProgramService.java:
```diff
@@ -65,8 +65,8 @@ public interface ReadOnlyApplicantProgramService {
 
-  /** Get the block with the given block ID */
-  Optional<Block> getBlock(String blockId);
+  /** Get the active block with the given block ID */
+  Optional<Block> getActiveBlock(String blockId);
```
**question (blocking):** clarify what "active" means. also clarify in the javadoc for this method what will happen if the `blockId` parameter is for a block that is not active.

azhangggggg replied:
added more comments in the javadoc explaining what `active` means and what will happy if the `blockId` parameter is for a block that is not active.

bion reviewed (changes requested):
- please either remove or update the release notes section of the PR description
- add the `ignore-for-release` tag since this does not change behavior

azhangggggg commented:
> * please either remove or update the release notes section of the PR description
> * add the `ignore-for-release` tag since this does not change behavior

removed the released notes section. added `ignore-for-release` tag.
